src/components/records/RecordsList.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import useResource from '../../hooks/useResource';
+
+interface Record {
+    id: string;
+    name: string;
+}
+
+interface ResourceListProps {
+    resource: string;
+}
+
+const ResourceList = ({ resource }: ResourceListProps) => {
```

Comment:
Please be consistent with the naming: Is this a resource or record list?

Reply:
What I had in mind is that a resource is "users" or "posts" for example, but a "record" is what you get when fetching from the resource. I realize it's a little confusing, how would you call each of those?